package.json

```diff
@@ -27,8 +27,9 @@
     "@axelar-network/axelar-gmp-sdk-solidity": "5.6.4",
     "@axelar-network/interchain-token-service": "1.2.1",
     "@cosmjs/cosmwasm-stargate": "^0.32.1",
-    "@ledgerhq/hw-transport-node-hid": "^6.27.21",
     "@ledgerhq/hw-app-eth": "6.32.2",
+    "@ledgerhq/hw-transport-node-hid": "^6.27.21",
+    "@openzeppelin/defender-relay-client": "^1.54.1",
```

Comment:
put it under dev dependency since we aren't exporting this script? want to reduce bloat